Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2227,6 +2216,26 @@ extension BrowserViewController: ContextMenuHelperDelegate {
     }
 }
 
+extension BrowserViewController {
+    func checkIfSwitchTabImmediately(tab: Tab) {
+        // Depending on user preference, new tab to be opened in background or be switched to immediately
+        // Default is to open in foreground
+        let switchToNewTabImmediately = self.profile.prefs.boolForKey(PrefsKeys.KeySwitchToNewTabImmediately) ?? true
```

Comment:
The current default behaviour is to open the tab in the background and show the ButtonToast. Can this be `?? false` instead.